metron-analytics/metron-statistics/src/test/java/org/apache/metron/statistics/StellarStatisticsFunctionsTest.java

```diff
@@ -356,6 +357,49 @@ public void testSkewness() throws Exception {
     assertEquals(stats.getSkewness(), (Double) actual, 0.1);
   }
 
+  @Test
+  public void testStatsBin() throws Exception {
```

Comment:
Given that it's not entirely obvious what this is doing and why it is a valid test, do you think it merits a comment with the above in the test?